mllib/src/main/scala/org/apache/spark/ml/param/params.scala

```diff
@@ -325,7 +379,7 @@ class ParamMap private[ml] (private val map: mutable.Map[Param[Any], Any]) exten
   }
 
   /**
-   * Make a copy of this param map.
+   * Creates a copy of this param map.
    */
   def copy: ParamMap = new ParamMap(map.clone())
```

Comment:
Do we like copy methods to use parentheses or not?  I think we're not completely consistent.